25_28_10/src/functions.js

```diff
@@ -54,3 +54,52 @@ function appendSelectHtml(callback, options, parentEl, defaultOptionName) {
 
   return selectEl;
 }
+
+const addCarSelect = (cars) => {
+  usersBlock.childNodes.forEach((element) => {
+    let markup = `<select class='selectJS'><option>${cars[0]?.brand}</option><option>${cars[1]?.brand}</option><option>${cars[2]?.brand}</option><option>${cars[3]?.brand}</option></select>`;
```

Comment:
очень длинная строка. 
```suggestion
    let markup = `<select class='selectJS'>
      <option>${cars[0]?.brand}</option>
      <option>${cars[1]?.brand}</option>
      <option>${cars[2]?.brand}</option>
      <option>${cars[3]?.brand}</option>
    </select>`;
```